tests/write_font_test.py

```diff
@@ -42,6 +42,40 @@ def test_keep_glyph_names(svgs, color_format, keep_glyph_names):
         assert ufo.glyphOrder != ttfont.getGlyphOrder()
 
 
+@pytest.mark.parametrize("svgs", [("rect.svg", "one-o-clock.svg")])
+@pytest.mark.parametrize(
+    "color_format",
+    [
+        "cff_colr_0",
+        "cff_colr_1",
+        "glyf_colr_0",
+        "glyf_colr_1",
+        "picosvg",
+    ],
+)
+@pytest.mark.parametrize("version_major", [12, 9])
+@pytest.mark.parametrize("version_minor", [0, "008"])
+# Test version 16.28
+def test_version_minor(svgs, color_format, version_major, version_minor):
```

Comment:
why do you parametrize the `version_major ` and `version_minor` if you then discard the parametrized value and hard-code 16.28 in the body of the test function?

Reply:
Done